SampleAuth.php

```diff
@@ -55,14 +62,29 @@ function auth_user_flags( $p_event_name, $p_args ) {
 			return null;
 		}
 
-		$t_access_level = user_get_access_level( $t_user_id, ALL_PROJECTS );
+		if( $t_user_id ) {
+		    $t_access_level = user_get_access_level( $t_user_id, ALL_PROJECTS );
 
-		# Have administrators use default login flow
-		if( $t_access_level >= ADMINISTRATOR ) {
+		    # Have administrators use default login flow
```

Comment:
Use tabs instead of spaces across the PR.

Reply:
I'm having tabs here in midnight commander, however, they are 4 spaces wide tabs.